### PR TITLE
 Make all components in the migration/replace functests fail

### PR DESCRIPTION
**Pull request by jschmid1**
$subject

-----------------

**Checklist:**
- [x] Added unittests and or functional tests
~- [ ] Adapted documentation~
~- [ ] Referenced issues or internal bugtracker~
- [ ] Ran integration tests successfully ( trigger with @susebot run teuthology )

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -176,19 +176,19 @@ copy-files:
 	install -d -m 755 $(DESTDIR)/srv/salt/ceph/functests/1node/tuned/off
```
Comment (smithfarm) [inline Makefile]: Could you fix the commit title? (Should be "move 1node.migrate to 3nodes.migrate", I guess.)

Comment (jschmid1) [reply Makefile]: err, yes

**Comment by jschmid1** [comment]
@susebot run teuthology suse/tier2/migration

**Comment by jschmid1** [comment]
> Commit 0903eb3 is NOT OK for suite suse/tier2/migration.

There is no counterpart in `https://github.com/SUSE/ceph/tree/ses6/qa/suites/suse/tier2` yet.

**Comment by jschmid1** [comment]
@susebot run teuthology suse/tier1/functional

**Comment by jschmid1** [comment]
@susebot run teuthology suse/tier2/migration

**Comment by jschmid1** [comment]
@susebot run teuthology suse/tier2/migration

**Comment by jschmid1** [comment]
lots of `KeyError: 'smoketests.pillar'`. Need to investigate

**Comment by jschmid1** [comment]
@susebot run teuthology suse/tier2/migration

**Comment by jschmid1** [comment]
I cannot reproduce the failures that we're seeing in the CI, retriggering

**Comment by jschmid1** [comment]
> The PR title and commit titles are...unsettling to say the least. I'd be happy about a change here. Somehting like "add failhard to all functests"?

I should've set the DNM flag, sorry. That's more of a test PR to figure out why migration tests are failing so horribly on OVH but not on local machines

File: srv/salt/_modules/osd.py [hunk 2]
```diff
@@ -1541,7 +1541,7 @@ def unmount(self):
                 else:
                     mount = entry[0]
                 if mount in mounted:
-                    cmd = "umount {}".format(mount)
+                    cmd = "umount -lf {}".format(mount)
```
[the remark this answers is not in the document]
Comment (jschmid1) [reply srv/salt/_modules/osd.py]: In our CI I saw multiple cases of a dangling mountpoint although the osd was destroyed. osd.remove should maybe more aggressive here. Hence the -l ( lazy ) and the -f ( force )

If this PR will improve things, although it doesn't look like it does, this change will go separately

**Comment by kshtsk** [comment]
@susebot run teuthology suse/tier2/migration

**Comment by kshtsk** [comment]
@susebot run teuthology suse/tier2/migration

**Comment by smithfarm** [comment]
@jschmid1 Ooops, it's `suse/tier3/migration` but I haven't migrated it yet. Give me a few minutes.

**Comment by smithfarm** [comment]
@susebot run teuthology deepsea/tier3/migration

**Comment by smithfarm** [comment]
```
2018-10-18T11:46:35.876 INFO:tasks.workunit.client.salt_master.target158069064243.stderr:+ sudo deepsea state.orch ceph.functests.3nodes.migrate
2018-10-18T11:46:36.550 INFO:tasks.workunit.client.salt_master.target158069064243.stderr:Usage: deepsea [OPTIONS] COMMAND [ARGS]...
2018-10-18T11:46:36.551 INFO:tasks.workunit.client.salt_master.target158069064243.stderr:
2018-10-18T11:46:36.551 INFO:tasks.workunit.client.salt_master.target158069064243.stderr:Error: No such command "state.orch".
```

@jschmid1 @rjfd I guess the correct form is `deepsea run state.orch $ORCH_ENTRYPOINT` ?

**Comment by smithfarm** [comment]
@susebot run teuthology deepsea/tier3/migration

**Comment by jschmid1** [comment]
This is an actual failure. Investigating

**Comment by jschmid1** [comment]
stale, closing

**Pull request closed without merging**